docs/installation.rst

```diff
@@ -10,11 +10,10 @@ Warnings
 
 .. warning:: Pillow >= 2.1.0 no longer supports "import _imaging". Please use "from PIL.Image import core as _imaging" instead.
 
-Notes
------
-
-.. note:: Pillow is supported on the following Python versions
+Python Support
+--------------
```

Comment:
```suggestion
Python Version Support
----------------------
```
Feel free to disregard this suggestion. My thinking is that this is not about whether or not we support Python, but about what version of Python we support.

Reply:
I think I'll keep it for brevity, and it should be ambiguous enough given the section contents and that this is a Python Imaging Library :) But thanks!